index.js

```diff
@@ -294,12 +312,17 @@ Use lotsOfArt to do the following:
 🌟 EXAMPLE: lotsOfArt(artists) will return ["Amedeo Modigliani", "Rene Magritte", ... "Albrecht Dürer"]
 */
 
-function lotsOfArt(/*Your Code Here*/) {
-  /*Your Code Here*/
+function lotsOfArt(array) {
```

Comment:
Awesome job here! Definitely a good use of a for loop. I would also recommend looking up other ways of solving this. Look into the array.filter function.

Reply:
The reason why I didn't use the array.filter method is because in the videos they didn't show us how to use it, and I was uncertain if I could use other methods than what was taught. 

Also, my brain apparently automatically goes to for loops when thinking about going through an array to find or do something.